src/common/components/loading/loading.component.js

```diff
@@ -36,6 +36,7 @@ class LoadingController {
   constructor () {
     this.type = 'block'
     this.inline = false
+    this.iconFirst = false
```

Comment:
Does this give the binding a default value? I'm not necessarily opposed, but I don't think it's necessary because we only add the `.icon-first` class if `$ctrl.iconFirst === 'true'`. This check evaluates the same whether or not `iconFirst` has a default value.

Reply:
I set it as false, so developers know it is a boolean and can quickly understand how to set the icon first.

Reply:
Got it. Since `@` bindings always create strings, do you think it would be better to set the default value to the string `"false"`? Or make it a `<` binding?

Reply:
That is a good point. Let's use `<`

Reply:
I went ahead switched it to `<`